kernel/arch/dreamcast/include/dc/matrix.h

```diff
@@ -93,9 +93,9 @@ void mat_apply(matrix_t *src);
     \param  invecs          The list of input vectors.
     \param  outvecs         The list of output vectors.
     \param  veccnt          How many vectors are in the list.
-    \param  vecskip         Number of empty bytes between vectors.
+    \param  stride          Number of bytes between vectors.
 */
-void mat_transform(vector_t *invecs, vector_t *outvecs, int veccnt, int vecskip);
+void mat_transform(vector_t *invecs, vector_t *outvecs, int veccnt, int stride);
```

Comment:
```suggestion
void mat_transform(vector_t *invecs, vector_t *outvecs, uint32_t veccnt, uint32_t stride);
```
Since the veccnt and stride cant be negative I think it would be good to update the types as well.

Reply:
Well, actually, might even be better to make `stride` a `size_t` here to denote something you'd typically use the `sizeof()` operator on, in byte units.

Reply:
Shit, nvm. If we change these argument types, we probably have to adjust the 'semblies, which is probably outside the scope of this PR.